docs/db-operations.md

```
@@ -112,3 +112,39 @@ pnpm wrangler d1 execute webull-trading-staging --env=staging --remote \
   --command "SELECT * FROM inverse_pairs"
 ```
 
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)